templates/dir.html.ep

```diff
@@ -83,18 +83,18 @@
       </tr>
       % for my $file (@$files) {
       <tr>
-        <td class="name break-long-content"><a href='<%= $file->{url} %>'><%== $file->{name} %></a></td>
       %   if ($file->{mtime}) {
-        <td class='size'><%= $file->{dir} ? '' : $file->{size} %></td>
-        <td class='mtime'><%= $file->{mtime} %></td>
+      %     my $colspan = $file->{dir} ? 'colspan="2" ' : '';
+        <td <%= $colspan %>class="name"><a href="<%= $file->{url} %>"><%== $file->{name} %></a></td>
+        <td class="size"><%= $file->{dir} ? '' : $file->{size} %></td>
+        <td class="mtime"><%= $file->{mtime} %></td>
       %   } else {
-        <td/><td/>
+      %     my $colspan = $file->{dir} ? 4 : 3;
+        <td colspan="<%= $colspan %>" class="name"><a href="<%= $file->{url} %>"><%== $file->{name} %></a></td>
       %   }
       %   if (!$file->{dir}) {
-        <td class='mirrors'><a href='<%= $file->{url} %>.metalink'>Details</a></td>
-      %   } else {
```

Comment:
It looks something is needed instead of these lines, because folder rows break too soon:
![image](https://user-images.githubusercontent.com/46994839/120456217-ad4d4d80-c395-11eb-9eb8-ab5ce8cdbe18.png)

Reply:
Looks like I added colspan to the wrong column